### DISCUSSION TITLE
Lepisorus

### DISCUSSION THREAD

**Discussion by crothfels** [pteridogroup/ppg#63] [General]
hi all -- I've been procrastinating on my homework and have left this to the very last minute, and am just now struggling with the competing Lepisorus proposals. It seems to me that both proposals are well argued but the main gap to me is the distinctiveness/stability of the smaller genera -- Lemmaphyllum, Paragramma, Neocheiropteris, etc. Personally, I think that lumping can be a disruptive to stability as is splitting and can result in a suboptimal classification, so if these genera are easily diagnosable, and have a strong history of use, I'd probably prefer to recognize a new genus, even if it is small and semi-cryptic, in order to be able to continue to recognize long-standing ones. However, if the boundaries between these genera, and between them and Lepisorus s.str., are vague, or not long-established, I'd probably prefer to lump everything in Lepisorus. Can anyone help me out?

**Comment by gonggashan88** [comment]
Hi Carl et al.,

Zhang et al. (2020) page 6 presents a key to these genera. In fact, the morphological distinctions of these taxa are also shown in Zhao et al. (2019) key (page 15-17). The major differences lie in the ranks in two papers: 8 genera (with one new) vs. 1 genus with a number of sections. In the field, with some experience you normally won't have any problems in recognizing these genera. Here is the comparison of the two classifications:

Lepisorus s.s. vs. Lepisorus s.l.
Ellipinema vs. Lepisorus sect. Pseudoclathrati
Lemmaphyllum vs. Lepisorus sect. Lemmaphyllum
Lepidomicrosorium vs. Lepisorus sect. Lepidomicrosorium
Neocheiropteris vs. Lepisorus sect. Neocheiropteris
Neolepisorus vs. Lepisorus sect. Neolepisorus
Paragramma vs. Lepisorus sect. Paragramma
Tricholepidium vs. Lepisorus sect. Tricholepidium

There were 7 existing genera and a number of species names under them, most of which have widely been used for decades. Now that these supraspecific taxa are morphologically diagnosable, phylogenetically deeply diverged, and geographically coherent (e.g., tropical Paragramma vs. subtropical others), why on the earth should one merge them all and make/accept a number of new combinations?

**Comment by crothfels** [comment]
thanks Libing! That key is helpful.
